platform/darwin/test/MGLExpressionTests.mm

```diff
@@ -1039,6 +1039,25 @@ - (void)testGenericExpressionObject {
 
 #pragma mark - Localization tests
 
+- (void)testv11Localization {
+    {
+        NSExpression *original = [NSExpression expressionWithFormat:@"mgl_coalesce({%K, %K})", @"name_en", @"name"];
+        NSExpression *expected = [NSExpression expressionWithFormat:@"mgl_coalesce:({name_en, name})"];
```

Comment:
This is the ideal result, but we current getn `mgl_coalesce:({mgl_coalesce:({name_en, name}), mgl_coalesce:({name_en, name})})` instead.